doc/rtmpwebhookauth.md

```diff
@@ -1,24 +1,28 @@
 # RTMP Webhook Authentication
 
-Incoming RTMP streams can be authenticated using Webhooks. To use it you need to implement own web server which will be called by Livepeer node when new RTMP stream received to check if that stream is allowed.
+Incoming RTMP streams can be authenticated using Webhooks. To provide these webhooks, node operators must implement their own web service / endpoint to be accessed only by the Livepeer node. As new RTMP streams appear, the Livepeer node will hit this endpoint to determine whether the given stream is allowed.
 
-To enable Webhook authentication functionality Livepeer node should be started with flag `-authWebhookUrl`, like this:
+To enable webhook authentication functionality, the Livepeer node should be started with the flag, `-authWebhookUrl` along with the webhook endpoint URL.
 
-`livepeer -authWebhookUrl http://ownserver/auth`
+For example:
 
-For every incoming RTMP stream Livepeer node will make `POST` request to `http://ownserver/auth` endpoint passing url to which RTMP request was made as JSON object.
+```console
+livepeer -authWebhookUrl http://ownserver/auth
+```
+
+For each incoming RTMP stream, the Livepeer node will make a `POST` request to the `http://ownserver/auth` endpoint, passing the URL of the RTMP request as JSON object.
 
-For example, if incoming RTMP request was made to `rtmp://livepeer.node:1935/something?manifestID=manifest`, Liverpeer node will pass this object in request to webhook:
+For example, if the incoming RTMP request was made to `rtmp://livepeer.node:1935/something?manifestID=manifest`, the Liverpeer node will provide the following object as a request to the webhook endpoint:
 
 ```json
 {
     "url": "rtmp://livepeer.node:1935/manifest"
 }
 ```
 
-Webhook server should respond with code 200 if it wants to allow RTMP stream. Response with any other code will lead to Livepeer node dropping RTMP stream.
+The webhook server should respond with HTTP status code `200` in order to authenticate / allow the RTMP stream. A response with a HTTP status code other than `200` will cause the Livepeer node to drop the RTMP stream.
```

Comment:
> authenticate / allow 

authenticate / authorize ?

> cause the Livepeer node to drop the RTMP stream.

"cause the Livepeer node to terminate the RTMP stream"

Reply:
`allow` changed to `authorize` in 65d6f015b7aa875e6d606161da115e1559610b8a

Reply:
I disagree with usage of the word "terminate" here.  No matter what `go-livepeer` wants, it will not succeed in terminating the incoming stream if it arrives via UDP.   Even if it arrives via TCP, many third party RTMP streamers will attempt reconnection in the event of a disconnect.  I think the word, "drop" is most appropriate here, but if you reiterate your preference fir the word, "terminate" then I will change it.

Reply:
I disagree with usage of the word "terminate" here.  No matter what `go-livepeer` wants, it will not succeed in terminating the incoming stream if it arrives via UDP.   Even if it arrives via TCP, many third party RTMP streamers will attempt reconnection in the event of a disconnect.  I think the word, "drop" is most appropriate here, but if you reiterate your preference for the word, "terminate" then I will change it.

Reply:
> it will not succeed in terminating the incoming stream if it arrives via UDP

We don't use UDP for ingest though?

> many third party RTMP streamers will attempt reconnection in the event of a disconnect

Could use the term "disconnect" if that works better

>  think the word, "drop" is most appropriate here

To me, "drop" has a more transitory connotation, eg dropped packets, dropped segments - a temporary hiccup that may occur without affecting the rest of the processing. Here, we're stopping / terminating the processing entirely, at least from the node's perspective. Also, since "dropping segments" is a thing we do, I want to stay as consistent and tight with terminology as possible.

The term "disconnect" works as well, but also open to other suggestions.

Reply:
ok ... done in 07c6ab17ff7cadad02544cb112ed2aa4052c97fd